www/docs/style/sass/pages/_mp.scss

```diff
@@ -461,6 +559,36 @@
       border-top: 1px solid $borders;
     }
   }
+
+  &.policy-votes > li {
+      padding-top: 1em;
+      padding-bottom: 1em;
+      padding-right: 7em;
+
+      @media (min-width: $medium-screen) {
+          padding-right: 10em; // make room for longer "Show full debate" button
+      }
+  }
+}
+
+.policy-vote--minor {
+    display: none;
```

Comment:
It's impossible to see minor votes if JavaScript is disabled. This should be visible by default, with a `.js `.policy-vote--minor` to hide it, and then to show the button to see all.